lib/jekyll-redirect-from/redirector.rb

```diff
@@ -79,8 +79,10 @@ def config_github_url(site)
       end
     end
 
-    def config_baseurl(site)
-      site.config.fetch('baseurl', nil)
+    def config_url(site)
+      url = site.config.fetch('url', nil) || ""
+      baseurl = site.config.fetch('baseurl', nil) || ""
+      File.join url, baseurl
```

Comment:
That breaks the tests. If the key is explicitly set to null, it returns null / not the default value, I think.